### DISCUSSION TITLE
Font update

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [amwx/FluentAvalonia#525] [Q&A]
I've notice that are some symbols missing from the [documentation](https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font) like `ShoppingCart`, `DocumentApproval` or `ReportDocument`

Are there any plans to update the Symbols for all the icons?

**Comment by robloo** [comment]
see here: https://github.com/amwx/FluentAvalonia/discussions/544#discussioncomment-9090294
